src/include/duckdb/function/scalar_function.hpp

```diff
@@ -124,6 +124,21 @@ class ScalarFunction : public SimpleFunction {
 		}
 	}
 
+	template <class TR, class OP> static scalar_function_t GetScalarIntegerUnaryFunctionFixedReturn(SQLType type) {
```

Comment:
Same for this function, perhaps move it as a static function to numeric.cpp?

Reply:
I also notice that `GetNumericUnaryAggregate` is unused.

Reply:
Done.